src/select/select.stories.ts

```diff
@@ -42,6 +42,22 @@ storiesOf("Select", module).addDecorator(
 		</ibm-select>
 	`
 	}))
+	.add("Light", () => ({
+		template: `
+		<ibm-select [theme]="'light'">
```

Comment:
Just a small change:
```suggestion
		<ibm-select theme="light">
```